docs/requirements.txt

```diff
@@ -5,4 +5,12 @@
 sphinx==4.5.0
 sphinx_rtd_theme==1.0.0
 readthedocs-sphinx-search==0.1.2
-myst-parser==0.18.0
\ No newline at end of file
+myst-parser==0.18.0
+
+# For building library
+setuptools
```

Comment:
don't do this....

Reply:
if you use `pip install -e .`
You don't need to specify build dependencies here.